### PR TITLE
panel toggle for view UI Editor

### LINKED ISSUE #2627: add toggle for panel in view UI editor

### PR DESCRIPTION
**Pull request by iantrich**
fixes https://github.com/home-assistant/home-assistant-polymer/issues/2627

### CODE REVIEW

**Comment by bramkragten** [comment]
Should we check if the view doesn't have more than 1 view?

**Comment by iantrich** [comment]
Do you mean card?

**Comment by bramkragten** [comment]
Yeah...

**Comment by iantrich** [comment]
How about just a general warning about only displaying the first card? Maybe even mention stacks? Or poiyto docs?

File: src/panels/lovelace/editor/view-editor/hui-view-editor.ts [hunk 1]
```diff
@@ -94,6 +102,12 @@ export class HuiViewEditor extends LitElement {
           .configValue="${"theme"}"
           @theme-changed="${this._valueChanged}"
         ></hui-theme-select-editor>
+        <paper-toggle-button
+          ?checked="${this._panel !== false}"
+          .configValue="${"panel"}"
+          @change="${this._valueChanged}"
+          >Panel Mode?</paper-toggle-button
```
[the remark this answers is not in the document]
Comment (bramkragten) [reply src/panels/lovelace/editor/view-editor/hui-view-editor.ts]: I would like to have everything in the UI, but agree with you, should we add an advanced option/toggle or something?

image

**Comment by balloob** [comment]
I am going to merge this. Add the warning and advanced options expansion (I like that idea!) in a new PR.

**Pull request merged**